variables.tf

```
@@ -98,6 +98,12 @@ variable "kind" {
   }
 }
 
+variable "application_insights_key" {
+  description = "The Instrumentation Key for connecting the Function App to Application Insights."
```

* Update description to be more similar to that of `application_insights_connection_string`.
* Update description to clarify that it's recommended to use `application_insights_connection_string` instead.

```suggestion
variable "application_insights_key" {
  description = "The instrumentation key of the Application Insights component to connect to this Function App. Support for instrumentation key ingestion will end on March 31, 2025. It's recommended to use application_insights_connection_string instead."
```

---

@kov117 Don't forget the commented reference! 🙂

---

right my bad 😅